sklearn/model_selection/_search.py

```diff
@@ -786,6 +792,7 @@ class GridSearchCV(BaseSearchCV):
             'test_mean_score'   : [0.81, 0.60, 0.75, 0.82],
             'test_std_score'    : [0.02, 0.01, 0.03, 0.03],
             'test_rank_score'   : [2, 4, 3, 1],
+            'test_mean_time'    : [--, -- ,-- ,--],
```

Comment:
Why is this masked? Just modify this to show some random times. (After we decide on what to add.)